### PR TITLE
irssi: bump

### PR DESCRIPTION
**Pull request by extrowerk**

### CODE REVIEW

File: net-irc/irssi/irssi-1.1.1.recipe [hunk 1]
```diff
@@ -4,14 +4,14 @@ It uses a text-mode user interface. Numerous Perl scripts have been \
...
 are available which add encryption and protocols such as ICQ and XMPP."
 HOMEPAGE="https://irssi.org/"
-COPYRIGHT="1999-2016 Timo Sirainen"
+COPYRIGHT="1999-2018 Timo Sirainen"
 LICENSE="GNU GPL v2"
 REVISION="1"
 SOURCE_URI="https://github.com/irssi/irssi/releases/download/$portVersion/irssi-$portVersion.tar.gz"
-CHECKSUM_SHA256="48ff359f8abb706e729cbb8bc7bd93863e4e785f9a05d993a398310ac74beea8"
+CHECKSUM_SHA256="ea36959398f8c3d8f52065818e3c1500908390dd5d1125ece1a382e77cb84925"
 SOURCE_DIR="irssi-$portVersion"
```
Comment (alaviss) [inline net-irc/irssi/irssi-1.1.1.recipe]: Isn't this the default?

```diff
-SOURCE_DIR="irssi-$portVersion"

File: net-irc/irssi/irssi-1.1.1.recipe [hunk 2]
```diff
@@ -4,14 +4,14 @@ It uses a text-mode user interface. Numerous Perl scripts have been \
 made available for Irssi to customise how it looks and operates. Plugins \
 are available which add encryption and protocols such as ICQ and XMPP."
 HOMEPAGE="https://irssi.org/"
-COPYRIGHT="1999-2016 Timo Sirainen"
+COPYRIGHT="1999-2018 Timo Sirainen"
```
Comment (alaviss) [inline net-irc/irssi/irssi-1.1.1.recipe]: There're more people than just Timo developing irssi. This should be updated to include all copyright holders.

File: net-irc/irssi/irssi-1.1.1.recipe [hunk 3]
```diff
@@ -4,14 +4,14 @@ It uses a text-mode user interface. Numerous Perl scripts have been \
...
 LICENSE="GNU GPL v2"
```
Comment (alaviss) [inline net-irc/irssi/irssi-1.1.1.recipe]: An exception was also [granted][0].

> Specific permission is granted for the GPLed code in this distribution to
be linked to OpenSSL without invoking GPL clause 2(b).


[0]: https://github.com/irssi/irssi/blob/1.1.1/COPYING#L1

Comment (alaviss) [reply net-irc/irssi/irssi-1.1.1.recipe]: The exception means that it's a customized version of GNU GPL v2, so please add a new license called "GNU GPL v2 with OpenSSL exception" and adjust `LICENSE` accordingly

**Comment by pulkomandy** [comment]
On Sat, Mar 10, 2018 at 08:19:53PM -0800, alaviss wrote:
> alaviss commented on this pull request.
> 
> 
> 
> > @@ -4,14 +4,14 @@ It uses a text-mode user interface. Numerous Perl scripts have been \
>  made available for Irssi to customise how it looks and operates. Plugins \
>  are available which add encryption and protocols such as ICQ and XMPP."
>  HOMEPAGE="https://irssi.org/"
> -COPYRIGHT="1999-2016 Timo Sirainen"
> +COPYRIGHT="1999-2018 Timo Sirainen"
>  LICENSE="GNU GPL v2"
> 
> The exception means that it's a customized version of GNU GPL v2, so please add a new license called "GNU GPL v2 with OpenSSL exception" and adjust `LICENSE` accordingly

Actually you could list that as two licenses:

GNU GPL v2
OpenSSL GPL2 exception

This way we don't need a second full copy of the GPL2.

**Pull request merged**